packages/components/src/Illustration/Scene/BrandMomentCaptureIntro/_docs/BrandMomentCaptureIntro.stickersheet.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { BrandMomentCaptureIntro } from "../index"
+
+export default {
+  title: "Components/Illustrations/BrandMomentCaptureIntro",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header headings={["Default", "Autoplay (hover)"]} />
+      <StickerSheet.Body>
+        <StickerSheet.Row>
+          <BrandMomentCaptureIntro />
+        </StickerSheet.Row>
+      </StickerSheet.Body>
+    </StickerSheet>
+  ),
+  parameters: {
+    pseudo: {
+      hover: true,
```

Comment:
I've blanketed this with a hover true for chromatic to snapshot the autoplay pause button on the video render of the illustration. 

I started going down the route of updating the props to allow for data attributes so the selector could be use for a hover state but we would need to update several types and how props are spread in its various shared subcomponents so I opted to leave this out (especially due to the snowflake nature of the component)

Reply:
Oh good spot, thanks